src/palindrom-dom.js

```diff
@@ -95,58 +94,60 @@ const PalindromDOM = (() => {
                 this.morphUrlEventHandler
             );
         }
-        
+
         /**
          * @param {String} href
-         * @throws {Error} network error if occured
-         * @fires Palindrom#palindrom-before-redirect 
-         * @fires Palindrom#palindrom-after-redirect 
-         * 
+         * @param {boolean} [throwOnCancellation=true] whether to throw an error when `palindrom-before-redirect`'s default behaviour is prevented using event.preventDefault(), defaults to `true`
+         * @throws {Error} network error if occured or the `palindrom-before-redirect` was cancelled by calling event.preventDefault()
+         * @fires Palindrom#palindrom-before-redirect
+         * @fires Palindrom#palindrom-after-redirect
+         * @returns {Response} response (https://github.com/axios/axios#response-schema)
          */
-        async getPatchUsingHTTP(href) {
+        async getPatchUsingHTTP(href, throwOnCancellation = true) {
```

Comment:
is something different then I expected. Why don't you add `.catch(...)` in morphUrl?
The `throwOnCancellation` argument is quite smelly to me. Why don't you catch the rejection it in `morphUrlEventHandler`, which is used by `palindrom-redirect-pushstate`, `palindrom-morph-url` event listeners.

`morphUrlEventHandler` though is not used by `clickHandler` and that's the handler where I could see the exception the most often. But I suppose you can change `clickHandler` to use `morphUrlEventHandler` and catch there.